src/roman_datamodels/maker_utils/_common_meta.py

```diff
@@ -444,7 +444,7 @@ def mk_mosaic_meta(**kwargs):
     dict (defined by the wfi_mosaic-1.0.0 schema)
     """
 
-    meta = {}
+    meta = mk_basic_meta(**kwargs)
```

Comment:
This will need removal and the appropriate keywords populated in the mosaic_basic maker utility.